packages/server/postgres/queries/getPatientZeroByDomain.ts

```diff
@@ -0,0 +1,10 @@
+import getPg from '../getPg'
+import IUser from '../types/IUser'
+import {getPatientZeroByDomainQuery} from './generated/getPatientZeroByDomainQuery'
+
+const getPatient0ByDomain = async (domain: string): Promise<IUser> => {
+  const usersInDomain = await getPatientZeroByDomainQuery.run({domain}, getPg())
+  return usersInDomain[0] as unknown as IUser
```

Comment:
+1 no need to do `as unknown`, as `IUser` will work just fine. also, if you do that, there's no need to specify the return type, ts will infer it. so you can also remove `Promise<IUser` from l5

Reply:
I got this error if not converting to `unknown` first:
<img width="592" alt="Screen Shot 2022-08-30 at 10 15 56 AM" src="https://user-images.githubusercontent.com/1879975/187500723-a8222a8d-db1c-40f3-9474-f7f8309eda13.png">
I think that's why we are doing it that way today, e.g., [here](https://github.com/ParabolInc/parabol/blob/master/packages/server/postgres/queries/getUsersByDomain.ts#L7)

Reply:
oh interesting, it worked just fine on my end. let's leave it as is then!